### ISSUE TITLE
Error identification and suggestion on foreground and background color inputs (WCAG SC 11.3.3.1 - 11.3.3.3)

### ISSUE THREAD

**Issue by laurencechavet** [ThePacielloGroup/CCAe#330]
**Describe the bug**
Visual users have neither error identifications in text nor suggestion of correction for both the foreground and background inputs.
Screen readers have no error identifications in text for the foreground input and no suggestion of correction for the background input.
The error message is in English even in the French page for both the foreground and background color inputs

**Desktop (please complete the following information):**
 - OS: Windows 10 64 bits
 - Version CCA 3.4.2

**To Reproduce**
- In the foreground input, type an incorrect format (e.g. "sss")
- In the developer tools, see that for text readers, there is an aria-live polite with the error message that is associated to the input through an aria-describedby. It is announced by NVDA when focusing out or focusing back in the input. So this is all good for non-visual users on 11.3.3.1. But visual users have only an alert icon mentioning the field has an error, and there is no text identifying the error. So the foreground input is missing
 -- A text error message for visual users
 -- An error suggestion for visual users and screen readers
 -- A text error in the appropriate language for screen readers
- Do the same input "sss" in the background color input
- In the developer tools, see that for text readers, there is an aria-live polite with the error message but it is NOT associated to the input so it is not read when focusing out or focusing back in the input. Only the supported format are announced as the help section is associated with aria-describedby, which is good for 11.3.3.3. But visual users have only an alert icon mentioning the field has an error, and there is no text identifying the error and no suggestions. So the background input is missing
 -- A text error message for visual users and screen readers
 -- An error suggestion for visual users
 -- A text error in the appropriate language for screen readers

**Expected behavior**
When the user input an wronf format in both the foreground and background inputs, the application must provide
- A text error message for visual users and screen readers (so in text, and associated to the input)
- An error suggestion for visual users and screen readers  
- A text error in the appropriate language for visual users and screen readers

**Screenshots**
![bug_11 3 3 1_ForegroundMissingErrorIdentificationInText](https://github.com/ThePacielloGroup/CCAe/assets/65306416/bc1ab558-c3f0-404d-8bed-a845d61dfa0e)
![bug_11 3 3 1_BackgroundMissingErrorIdentificationInText](https://github.com/ThePacielloGroup/CCAe/assets/65306416/e7a6b272-10bf-49f0-b676-74e137b23129)
![bug_11 3 3 1_ForegroundENErrorIdentificationInFR](https://github.com/ThePacielloGroup/CCAe/assets/65306416/86a7f09d-5339-4187-af8f-22f050b2e183)
![bug_11 3 3 1_BackgroundENErrorIdentificationInFR](https://github.com/ThePacielloGroup/CCAe/assets/65306416/5896e4bc-486c-4653-a015-80d16763c30d)

**Additional context**
N/A

**Comment by laurencechavet** [comment]
Hi, Any news on this bug. Thanks
